bin/alluxio

```diff
@@ -20,7 +20,7 @@ function printUsage {
   echo -e "  formatWorker  \t Format Alluxio worker storage"
   echo -e "  bootstrapConf \t Generate a config file if one doesn't exist"
   echo -e "  fs            \t Command line tool for interacting with the Alluxio filesystem."
-  echo -e "  fsadmin       \t Command line tool for admins to manage the Alluxio filesystem."
+  echo -e "  fsadmin       \t Command line tool for interacting with the Alluxio filesystem admins."
```

Comment:
nit: for use by Alluxio filesystem admins?

Reply:
make sense. fixed, thanks